tests.py

```diff
@@ -920,8 +932,14 @@ def test_wrapInstance():
         finally:
             app.exit()
 
-    def test_implicit_wrapInstance():
-        """.wrapInstance doesn't need the `base` argument"""
+    def test_implicit_wrapInstance_for_base_types():
+        """Tests implicit `Foo` -> `Foo` cast of `Foo` pointer to `Foo` object,
```

Comment:
This summary is a little long, and has a line-break.

The first line of the docstring is what shows up as [the name of the test](https://travis-ci.org/github/mottosso/Qt.py/jobs/724576499#L576) so we'd better keep it short and sweet. You can move the elaboration to the next line and beyond. Looks pretty good already perhaps, without the comma at the end there.